.github/workflows/integration-test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.3.0/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-2.3.0-linux-x64.tar.gz
           tar -xzf opensearch-*.tar.gz
           rm -f opensearch-*.tar.gz
-          
+
```

Comment:
Please remove this change.

Reply:
Ya I have reset it to main and pushed again. Thx for the review @cliu123!